spec/models/snap_application_attributes_spec.rb

```diff
@@ -50,6 +50,47 @@
         self_employed_household_members_no: "Yes",
         employed_household_members_yes: nil,
         employed_household_members_no: "Yes",
+        additional_income_yes: nil,
+        additional_income_no: "Yes",
+        additional_income_social_security: nil,
+        additional_income_pension: nil,
+        additional_income_unemployment: nil,
+        additional_income_ssi: nil,
+        additional_income_workers_compensation: nil,
+        additional_income_child_support: nil,
+        additional_income_other: nil,
+        court_ordered_expenses_child_support: nil,
+        court_ordered_expenses_alimony: nil,
+        court_ordered_expenses_yes: nil,
+        court_ordered_expenses_no: "Yes",
+        monthly_court_ordered_expenses: nil,
+        medical_expenses_yes: nil,
+        medical_expenses_no: "Yes",
+        medical_expenses_health_insurance: nil,
+        medical_expenses_prescriptions: nil,
+        medical_expenses_dental: nil,
+        medical_expenses_transportation: nil,
+        medical_expenses_hospital_bills: nil,
+        medical_expenses_other: nil,
+        monthly_medical_expenses: nil,
+        dependent_care_yes: nil,
+        dependent_care_no: "Yes",
+        care_expenses_childcare: nil,
+        care_expenses_disabled: nil,
+        monthly_care_expenses: nil,
+        rent_expense_yes: nil,
+        rent_expense: nil,
+        property_tax_expense_yes: nil,
+        annual_property_tax_expense: nil,
+        annual_insurance_expense: nil,
+        insurance_expense_yes: nil,
+        utility_heat: nil,
+        utility_cooling: nil,
+        utility_electricity: nil,
+        utility_water_sewer: nil,
+        utility_trash: nil,
+        utility_phone: nil,
+        utility_other: nil,
```

Comment:
I know! Some of these are booleans in the DB and others are arrays. Perhaps will try to standardize a bit...